### ISSUE TITLE
BUG: screener/base: FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated.

### ISSUE THREAD

**Issue by deeleeramone** [lit26/finvizfinance#118]
### finvizfinance version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the master branch of finvizfinance.

### Reproducible Example

With Pandas 2.2.2, a FutureWarning is broadcast when using the `screener_view()` method.

```
.../python3.11/site-packages/finvizfinance/screener/base.py:131: FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated. In a future version, this will no longer exclude empty or all-NA columns when determining the result dtypes. To retain the old behavior, exclude the relevant entries before the concat operation.
  return pd.concat([df, pd.DataFrame(frame)], ignore_index=True)
```

### Issue Description

Same as above; doesn't break anything but future versions of Pandas may.

### Expected Behavior

Before `pd.concat`, drop all-NA columns and/or check if df.empty before concatenation.

### Installed Versions

1.0.1

**Comment by lit26** [comment]
Yeah, I am trying to find a good way to fix this. Will be fixed soon.

**Comment by deeleeramone** [comment]
Keep up the good work!

I'm working on an integration of the screener, using .ini files for presets.

![Screenshot 2024-07-27 at 2 03 10 PM](https://github.com/user-attachments/assets/49f0293d-570a-4ad2-976f-d9ef61ccd061)

![Screenshot 2024-07-27 at 12 24 14 PM](https://github.com/user-attachments/assets/8fe5c8e2-d03d-4f85-8582-8e9e30034cc1)

**Comment by lit26** [comment]
The warning is fixed in v1.1.0. Thanks for reporting.
